cv-19-i-need-help-test/EndToEndTests/V2/GetHelpRequestTest.cs

```diff
@@ -0,0 +1,122 @@
+using System;
+using System.Collections.Generic;
+using Amazon.Lambda.APIGatewayEvents;
+using AutoFixture;
+using CV19INeedHelp.Boundary.V2;
+using CV19INeedHelp.Boundary.V2.Responses;
+using CV19INeedHelp.Data.V1;
+using CV19INeedHelp.Helpers.V2;
+using CV19INeedHelp.Models.V1;
+using CV19INeedHelpTest.TestHelpers;
+using FluentAssertions;
+using Newtonsoft.Json;
+using NUnit.Framework;
+
+namespace CV19INeedHelpTest.EndToEndTests.V2
+{
+    [TestFixture]
+    public class GetHelpRequestTest
+    {
+        private string _currentConnStr;
+        private Handler _handler;
+        private Fixture _fixture;
+        private Cv19SupportDbContext _dbContext;
+
+        [SetUp]
+        public void SetUp()
+        {
+            _currentConnStr = Environment.GetEnvironmentVariable("CV_19_DB_CONNECTION");
```

Comment:
There is now a test setup class that you can inherit here so that you don't need this initial setup of the database. (Like here https://github.com/LBHackney-IT/cv-19-i-need-help-api/blob/282dd9596f438e7ab9c0f612d6dbef856b3850b3/cv-19-i-need-help-test/EndToEndTests/V2/GetHelpRequestsTest.cs#L17)

You'll need to change `_dbContext` to `DbContext` afterwards

Reply:
Changes applied in #34